lib/reporting/drop_off_report.rb

```diff
@@ -243,34 +230,21 @@ def dropoff_metrics_table
             denominator: idv_started,
           ),
         ],
-      ]
-    end
-
-    # rubocop:disable Layout/LineLength
-    def proofing_definition_table
-      [
-        ['Term', 'Description', 'Definition', 'Calculated'],
-        [
-          'Blanket Proofing',
-          'Full funnel: People who started proofing from welcome screen, successfully got verified credential and encrypted account',
-          'Percentage of users that successfully proofed over the total number of users that began the proofing process',
-          'Steps: "Verified" divided by "User agreement"',
-        ],
-        [
-          'Actual Proofing',
-          'Proofing funnel: People that submit and get verified',
-          'Percentage of users who submitted documents, passed instant verify and phone finder',
-          'Steps: "Encrypt account: enter password" divided by "Document submitted"',
-        ],
         [
-          'Verified Proofing',
-          'Proofing + encryption: People that get verified, encypt account and are passed back to Service Provider',
-          'Number of users who submitted documents, passed instant verify and phone finder, encrypted account, and sent to consent screen for sharing data with Service Provider',
-          'Steps: "Verified" divided by "Document submitted"',
+          'Workflow Complete - Total Pending',
```

Comment:
sorry, i've gotten a bit confused at the state of all this. 

i *think* this table should be removed entirely (as it was originally just a definition table for fields we're no longer using), and the workflow complete should probably just be an event in the existing table. does that make sense?

Reply:
I don’t see that as part of the ticket.

Reply:
This is the only part of the ticket about removing things

> Remove legacy percentages from the report including the code (i.e., blanket, actual, and verified)

and that isn’t the definitions portion of the table.

Reply:
I really don’t think the PR review is the time to add ACs to the ticket.

Reply:
> and that isn’t the definitions portion of the table.
thanks, you're right, I was confused by the git diff, you did delete the definitions portions of the table! thanks for clarifying

however, i disagree with `I really don’t think the PR review is the time to add ACs to the ticket.` code review is a perfect time to make sure software makes sense, whether or not it aligns perfectly with the ticket. (in many cases requiring follow-up with the product folks.) it's worth it to double check if we see something that doesn't seem to make sense. in this case, i was wrong, but that doesn't mean it's not the right time to ask the question.